playbooks/common/openshift-checks/README.md

```diff
@@ -36,8 +28,14 @@ against your inventory file. Here is the step-by-step:
     $ cd openshift-ansible
     ```
 
-2. Run the playbook:
+2. Run the appropriate playbook:
+
+    ```console
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/pre-install.yml
+    ```
+
+    or
 
     ```console
-    $ ansible-playbook -i <inventory file> playbooks/byo/openshift-preflight/check.yml
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/health.yml
```

Comment:
I was wondering the same... the health check playbooks don't seem to fully fit the others under `common/`... although they don't fully fit under `byo/` either.

Brainstorming a bit here but: what about moving to `playbooks/openshift-checks/`? and move the playbooks from `certificate_expiry` in there too?